### PR TITLE
LEAN-2915: CTC-010 - Styling of the Inspector History tab - FE implem…

### PR DESCRIPTION
**Pull request by mvidtman**
…entation

### CODE REVIEW

File: src/theme.ts [hunk 1]
```diff
@@ -80,9 +80,11 @@ interface Variations {
 interface Background {
   dark: string
   selected: string
+  newDesign?: States
```
Comment (mbartenev-atypon) [inline src/theme.ts:83]: Please change the name to more meaningful

Comment (mbartenev-atypon) [inline src/theme.ts:83]: Please change the name to more meaningful

File: src/theme.ts [hunk 2]
```diff
@@ -80,9 +80,11 @@ interface Variations {
...
 }
 interface Border {
   field: States
+  newDesign?: States
```
Comment (mbartenev-atypon) [inline src/theme.ts:87]: Please change the name to more meaningful

**Review by mbartenev-atypon — CHANGES_REQUESTED** [review]

File: src/theme.ts (hunk 1 above)
Comment (mvidtman) [reply src/theme.ts]: changed to 'tracked'

**Review by mbartenev-atypon — APPROVED** [review]

**Pull request merged**